cmd/dart-config/config.go

```diff
@@ -8,15 +8,16 @@ import (
 
 // Deployment documents an individual DART operational period with optional de-tiding parameters.
 type Deployment struct {
-	Network   string    `json:"network"`
-	Buoy      string    `json:"buoy"`
-	Location  string    `json:"location"`
-	Latitude  float64   `json:"latitude"`
-	Longitude float64   `json:"longitude"`
-	Depth     float64   `json:"depth"`
-	Detide    *Detide   `json:"detide,omitempty"`
-	Start     time.Time `json:"start"`
-	End       time.Time `json:"end"`
+	Network    string        `json:"network"`
+	Buoy       string        `json:"buoy"`
+	Location   string        `json:"location"`
+	Latitude   float64       `json:"latitude"`
+	Longitude  float64       `json:"longitude"`
+	Depth      float64       `json:"depth"`
+	Detide     *Detide       `json:"detide,omitempty"`
+	Correction time.Duration `json:"correction,omitempty"`
```

Comment:
Suggestion: `TimingCorrection`

why: The existence of the `Correction` struct that contains a `Timing` field is confusing.

Reply:
Yup, fixed as suggested.